src/components/landingPage/LandingPage.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
+import '../../assets/App.scss';
+import '../../assets/App.css';
```

Comment:
If this is App.css or scss, why isn't it available to the whole application. Do we have to import it in every other component as well?